src/display/api.js

```diff
@@ -1035,6 +1035,11 @@ var PDFPageProxy = (function PDFPageProxyClosure() {
       function operatorListChanged() {
         if (intentState.operatorList.lastChunk) {
           intentState.opListReadCapability.resolve(intentState.operatorList);
+
+          var i = intentState.renderTasks.indexOf(opListTask);
+          if (i >= 0) {
+            intentState.renderTasks.splice(i, 1);
+          }
```

Comment:
Looking at https://github.com/mozilla/pdf.js/blob/master/src/display/api.js#L1176, we probably want `intentState.receivingOperatorList = false;` here too.